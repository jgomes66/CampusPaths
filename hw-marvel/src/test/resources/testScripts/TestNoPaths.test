# Simple test to see what happens when there is no path
# between any of the existing characters

LoadGraph g noPaths.csv

ListChildren g A
ListChildren g B
ListChildren g C

FindPath g A B
FindPath g B A
FindPath g B C
FindPath g C B
FindPath g C A

ListNodes g
