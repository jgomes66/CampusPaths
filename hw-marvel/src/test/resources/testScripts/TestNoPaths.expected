# Simple test to see what happens when there is no path
# between any of the existing characters

loaded graph g

the children of A in g are:
the children of B in g are:
the children of C in g are:

path from A to B:
no path found
path from B to A:
no path found
path from B to C:
no path found
path from C to B:
no path found
path from C to A:
no path found

g contains: A B C
